BootloaderCommonPkg/Include/Library/IasImageLib.h

```diff
@@ -65,7 +65,7 @@ typedef struct {        /* a file (sub-image) inside a boot image */
 //
 typedef struct {
   UINT32  Signature;
-  UINT8 PubKeyMod[RSA_MOD_SIZE];
+  UINT8 PubKeyMod[RSA_MOD_SIZE_2048];
```

Comment:
RSA_MOD_SIZE_MAX? How to handle RSA_MOD_SIZE_3072?

Reply:
It would be modified while adding RSA3072 support

Reply:
Then, can you put the changes for hard-code fix only in this PR and make a separate PR for RSA3072?

Reply:
I think this is for IAS image format only, which will be deprecated.  So we only support RSA2048 for IAS.  To support new RSA3072, it is required to move to container.

Reply:
agreed with Maurice.

Reply:
Okay. Understood.